ftw/publisher/sender/taskqueue/queue.py

```diff
@@ -44,28 +48,28 @@ def __call__(self):
         filepath = self.request.form['filepath']
         path = self.request.form['path']
         additional_data = encode_after_json(json.loads(self.request.form['additional_data']))
-
         obj = api.portal.get().unrestrictedTraverse(path, None)
-
         require_token = self.request.form['token']
+        attempt = self.request.form['attempt']
+        current_token = IAnnotations(obj).get(TOKEN_ANNOTATION_KEY, None)
```

Comment:
`obj` may be `None` and this line depends on `obj` being an object. I suggest moving this line beneath the `if obj is None:` block.

Reply:
🤦‍♂️  Nice catch. I have fixed this.